### ISSUE TITLE
Missing characters and incorrect rendering of sequences of DIN 91379

### ISSUE THREAD

**Issue by vk-github18** [IBM/plex#399]
The list of characters and sequences according to (draft) DIN 91379 resp. DIN SPEC 91379.
[latin_letters_all.txt](https://github.com/IBM/plex/files/7027423/latin_letters_all.txt)
[latin_list_all.txt](https://github.com/IBM/plex/files/7027427/latin_list_all.txt)
is rendered as follows with IBMPlexSans-Regular with hb-view 2.6.8 under Linux.
[latin_letters_all_IBMPlexSans-Regular.pdf](https://github.com/IBM/plex/files/7027437/latin_letters_all_IBMPlexSans-Regular.pdf)
[latin_list_all_IBMPlexSans-Regular.pdf](https://github.com/IBM/plex/files/7027438/latin_list_all_IBMPlexSans-Regular.pdf)

There are missing characters and misplaced accents.

For reference I add the rendering with Noto-Sans
[latin_letters_all_NotoSans-Regular.pdf](https://github.com/IBM/plex/files/7027444/latin_letters_all_NotoSans-Regular.pdf)
[latin_list_all_NotoSans-Regular.pdf](https://github.com/IBM/plex/files/7027445/latin_list_all_NotoSans-Regular.pdf)
and for combining comma above right the rendering of Liberation Sans:
[combining_comma_above_right_LiberationSans-Regular.pdf](https://github.com/IBM/plex/files/7027448/combining_comma_above_right_LiberationSans-Regular.pdf)

The tested sequences are used in the following specification(DIN SPEC and draft DIN):

DIN SPEC 91379: Characters in Unicode for the electronic processing of names and dataexchange in Europe; with digital attachment

https://www.xoev.de/downloads-2316#StringLatin
https://www.din.de/de/wdc-beuth:din21:301228458

Draft DIN 91379: Characters and defined character sequences in Unicode for the electronic processing of names and data exchange in Europe, with CD-ROM

https://www.din.de/de/mitwirken/normenausschuesse/nia/entwuerfe/wdc-beuth:din21:339812645

See also:
googlefonts/noto-fonts#1882
https://github.com/liberationfonts/liberation-fonts/issues/45

**Comment by BoldMonday** [comment]
Thanks for the detailed report. Since this concerns quite a large amount of glyphs (I count approx. 150 that need to be added) and given that the full Plex family consists of 64 individual styles in total it might take a while to fulfill this request.

**Comment by vk-github18** [comment]
I agree, it is a big effort, to add the missing glyphs and check and correct the position of the accents.
On the other hand with conformance to the standard many languages written with Latin letters will be supported,
so IMHO it is worth the effort.
